content/applications/finance/expenses/reinvoice_expenses.rst

```diff
@@ -0,0 +1,144 @@
+
+.. _expenses/reinvoice-create:
+
+Create an expense
+=================
+
+First, when :doc:`creating a new expense <../expenses/log_expenses>`, the correct information needs
+to be entered to re-invoice the expense to a customer. Using the drop-down menu, select the
+:abbr:`SO (Sales Order)` to add the expense to in the :guilabel:`Customer to Reinvoice` field.
+
+Once a :abbr:`SO (Sales Order)` is attached to an expense report, and the report is saved, a
+:icon:`fa-money` :guilabel:`Sales Orders` smart button appears at the top-center of the screen.
+
+Next, select the :guilabel:`Analytic Distribution` the expense is posted to. Multiple accounts can
+be selected, if desired.
+
+To add another :guilabel:`Analytic Distribution`, click on the line to reveal the
+:guilabel:`Analytic` pop-over window. Click :guilabel:`Add a line`, then select the next
+:guilabel:`Analytic Distribution` from the drop-down field. If selecting more than one
+:guilabel:`Analytic Distribution`, the :guilabel:`Percentage` fields **must** be modified. By
+default, both fields are populated with `100%`. Adjust the percentages for all the fields, so the
+total is `100%`.
+
+.. example::
+   A painting company agrees to paint an office building that houses two different companies. During
+   the estimate, a meeting is held at the office location to discuss the project and view the
+   building.
+
+   Both companies agree to pay for the travel expenses for the painting company employees. When
+   creating the expenses for the mileage and hotels, **both companies** are listed in the
+   :guilabel:`Analytic Distribution` line, for 50% each.
+
+.. _expenses/reinvoice-report:
+
+Create an expense report
+========================
+
+After the expenses are created, the expense report must be :ref:`created <expenses/create_report>`
+and :ref:`submitted <expenses/submit>`, in the same manner as all other expenses.
+
+.. image:: reinvoice-expenses/reinvoice-expense.png
+   :align: center
+   :alt: Ensure the customer to be invoiced is called out on the expense.
+
+.. important::
+   Selecting the proper :abbr:`SO (Sales Order)` in the :guilabel:`Customer to Reinvoice` field is
+   **critical**, since this is how expenses are automatically invoiced after an expense report is
+   approved.
+
+   The :guilabel:`Customer to Reinvoice` field can be modified *only* until an expense report is
+   **approved**. After an expense report is approved, the :guilabel:`Customer to Reinvoice` field is
+   no longer able to be modified.
+
+.. _expenses/reinvoice-approve:
+
+Approve and post expenses
+=========================
+
+Before :doc:`approving an expense report <../expenses/approve_expenses>`, ensure the
+:guilabel:`Analytic Distribution` section is populated for every expense line.
+
+If an :guilabel:`Analytic Distribution` entry is missing, assign the correct accounts from the
+drop-down menu, then click :guilabel:`Approve`.
```

Comment:
Do you think it's important to mention that this 'Approve' button only appears after clicking 'Submit to manager'? I think that would be very helpful...